app/MMB/src/components/exploreCity/ExploreText.js

```diff
@@ -17,32 +20,39 @@ const ExploreText = (props: Props) => {
   const city = idx(props.data, _ => _.arrival.airport.city.name);
   const date = new Date(idx(props.data, _ => _.departure.time) || 0);
   const daysLeft = DateUtils.diffInDays(date, DateUtils.getUTCToday());
+  const airport = idx(props.data, _ => _.arrival.airport);
   return (
     <React.Fragment>
-      <Text style={styles.title}>
-        {daysLeft > 0 ? (
-          <Translation id="mmb.main_menu.explore_city.prepare_for_trip" />
-        ) : (
-          <Translation
-            id="mmb.main_menu.explore_city.spending_time_in"
-            values={{ city }}
-          />
-        )}
-      </Text>
-      <Text style={styles.subText}>
-        {daysLeft > 0 ? (
-          <Translation
-            id="mmb.main_menu.eplore_city.your_trip_in_days"
-            values={{
-              city,
-              days: daysLeft,
-            }}
-          />
-        ) : (
-          <Translation id="mmb.main_menu.explore_city.find_stuff_to_do" />
-        )}
-      </Text>
-      <ExploreButton cityName={city || ''} />
+      <View style={styles.padding}>
+        <Text style={styles.title}>
+          {daysLeft > 0 ? (
+            <Translation id="mmb.main_menu.explore_city.prepare_for_trip" />
+          ) : (
+            <Translation
+              id="mmb.main_menu.explore_city.spending_time_in"
+              values={{ city }}
+            />
+          )}
+        </Text>
+        <Text style={styles.subText}>
+          {daysLeft > 0 ? (
+            <Translation
+              id="mmb.main_menu.eplore_city.your_trip_in_days"
+              values={{
+                city,
+                days: daysLeft,
+              }}
+            />
+          ) : (
+            <Translation id="mmb.main_menu.explore_city.find_stuff_to_do" />
+          )}
+        </Text>
+      </View>
+      <View style={[styles.padding, styles.buttonContainer]}>
+        <ExploreButton cityName={city || ''} />
+      </View>
+      <SeparatorFullWidth />
+      <ExploreMenuGroup data={airport} />
```

Comment:
It seems like this should be concerning the airport of the first segment, e.g. on trip Vienna-Birmingham with segments Vienna-Dublin, Dublin-Birmingham, the explore menu group has button 
`Order Transport to DUB`.

Reply:
You are right, this should be arrival, not the actual leg. I will fix it tomorrow.

Reply:
I am unable to reproduce what you are mentioning. See screenshot. 
![image](https://user-images.githubusercontent.com/7705041/43758239-c4f9d940-9a1b-11e8-8fd4-a1e9c3814a33.png)

Reply:
And this is what I see in the real app:
![image from ios](https://user-images.githubusercontent.com/22741774/43760266-d8c5d8ce-9a21-11e8-8159-befe515b691d.png)